/colossus_subscriber_laser_scan_publisher:
  ros__parameters:
    azimuth_offset: 0
    range_offset: 0.0
    end_azimuth: 400
    end_bin: 2856
    power_threshold: 110
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    start_azimuth: 0
    start_bin: 0
    use_sim_time: false
